File: atcoder/abc072C_together.nim
# unihernandez22
# https://atcoder.jp/contests/abc072/tasks/arc082_a
# greedy

import tables, strutils, sequtils

discard stdin.readLine
var a = stdin.readLine.split.map(parseInt)

var possibles_x = initCountTable[int]()

for i in a:
  possibles_x.inc(i)
  possibles_x.inc(i+1)
  possibles_x.inc(i-1)

echo possibles_x.largest[1]

File: atcoder/abc072C_together.py
# Vicfred
# https://atcoder.jp/contests/abc072/tasks/arc082_a
# greedy, bucket sort
class Counter(dict):
    def __missing__(self, i):
        return 0

n = int(input())
a = list(map(int, input().split()))

counter = Counter()

for item in a:
    counter[item] += 1
    counter[item+1] += 1
    counter[item-1] += 1

maxima = max(counter.values())

print(maxima)

